### ISSUE TITLE
Updating dates to null is broken

### ISSUE THREAD

**Issue by seanmcne** [seanmcne/Microsoft.Xrm.Data.PowerShell#248]
Hey @kenakamu wasn't there a fix in 2.6 to address this? 

```powershell
Set-CrmRecord -EntityLogicalName contact -Id $conts.CrmRecords[0].contactid -Fields @{"birthdate"=$
null}
MapFieldTypeByFieldValue : Cannot bind argument to parameter 'Value' because it is null.
At C:\Users\seanmcn\Documents\WindowsPowerShell\Modules\Microsoft.Xrm.Data.Powershell\2.6\Microsoft.Xrm.Data.Powershell
.psm1:722 char:62
+ ...         $newfield.Type = MapFieldTypeByFieldValue -Value $field.Value
+                                                              ~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [MapFieldTypeByFieldValue], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,MapFieldTypeByFieldValue

Object reference not set to an instance of an object.
At C:\Users\seanmcn\Documents\WindowsPowerShell\Modules\Microsoft.Xrm.Data.Powershell\2.6\Microsoft.Xrm.Data.Powershell
.psm1:745 char:9
+         throw $conn.LastCrmException
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], NullReferenceException
    + FullyQualifiedErrorId : Object reference not set to an instance of an object.
```

**Comment by kenakamu** [comment]
Ah it may only work for entity reference then? Let me look into it

**Comment by seanmcne** [comment]
👍thanks!

**Comment by seanmcne** [comment]
Closing as this should be resolved in #266 and released in our next update
